src/v2i-hub/ImmediateForwardPlugin/src/ImmediateForward.cpp

```diff
@@ -1,17 +0,0 @@
-/*
- * ImmediateForward.cpp
- *
- *  Created on: Feb 26, 2016
- *      Author: ivp
- */
-
-#include "ImmediateForwardPlugin.h"
-
```

Comment:
Why do we remove the main function?

Reply:
I just moved it to the plugin class. I figured it is more concise. Many of our other plugins do the same